src/commands/tag.js

```diff
@@ -0,0 +1,114 @@
+import { writeObject } from './writeObject'
+import { readObject } from './readObject'
```

Comment:
For tree-shaking perf, we should use GitObjectManager directly because bringing in writeObject and readObject brings with it the parsers/serializers for every possible git object.